app/views/mentee_profiles/show.html.slim

```diff
@@ -7,21 +7,21 @@
        ='  @user.first_name 
        ='  @user.last_name
       p 
-       | From 
+       | From   
        = @user.country
       p
-        | Currently in: 
-        = @user.program_country
-        a.left10 = @user.timezone
+        | Currently in:   
+        = @user.program_country     
+        a.left10  = @user.timezone
   h3 Project
-  span.project-title = @project.title
+  span.project-title = @project.title if @project.title?
```

Comment:
if @project is nil it will still give you an error - and everywhere else you use similar if ;)